Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1917,6 +1942,28 @@ private static TimeInForce ConvertTimeInForce(string timeInForce)
             }
         }
 
+        private static DateTime ParseExpiryDateTime(string expiryDateTime)
+        {
+            // NOTE: we currently ignore the time zone in this method for a couple of reasons:
+            // - TZ abbreviations are ambiguous and unparsable to a unique time zone
+            //   see this article for more info:
+            //   https://codeblog.jonskeet.uk/2015/05/05/common-mistakes-in-datetime-formatting-and-parsing/
+            // - IB seems to also have issues with Daylight Saving Time zones
+            //   Example: an order submitted from Europe with GoodTillDate property set to "20180524 21:00:00 UTC"
+            //   when reading the open orders, the same property will have this value: "20180524 23:00:00 CET"
+            //   which is incorrect: should be CEST (UTC+2) instead of CET (UTC+1)
+
+            // We can ignore this issue, because the method is only called by GetOpenOrders.
```

Comment:
great comment above about IB's mishandling of time zones, perhaps add an additional blurb to the bottom line here. _why_ is it OK that it's only called by `GetOpenOrders`? -- my hunch is that it's because we only call `GetOpenOrders` during live trading, which means we won't be simulating TIF and instead will rely on brokerages to apply TIF properly.

Reply:
> it's because we only call GetOpenOrders during live trading, which means we won't be simulating TIF and instead will rely on brokerages to apply TIF properly.
This is exactly the reason -- I'll add this comment to make it more clear